extensions-core/google-extensions/src/test/java/org/apache/druid/storage/google/GoogleDataSegmentKillerTest.java

```diff
@@ -54,8 +51,7 @@ public class GoogleDataSegmentKillerTest extends EasyMockSupport
   private static final long TIME_0 = 0L;
   private static final long TIME_1 = 1L;
   private static final int MAX_KEYS = 1;
-  private static final Exception RECOVERABLE_EXCEPTION = new HttpResponseException.Builder(429, "recoverable", new HttpHeaders()).build();
```

Comment:
why is the test case for this scenario removed?

Reply:
delete can only throw runtime exceptions henceforth, which has implications on possible exceptions with its caller functions.

Reply:
but the library can still throw 429? are we no longer retrying for that particular HTTP response code? If we do, what test tests that.

Reply:
No the library can't throw 429 "checked" exception. All it can throw now is runtime "unchecked" exception which is non-retryable in our current logic. THe RECOVERABLE and NON-RECOVERABLE declared previously were internally both checked exceptions

Reply:
The new library throws StorageException, which, while not a checked exception, still need to be caught so that we can retry like we did earlier.

Reply:
Corrected.